analysis/study_definition.py

```diff
@@ -65,10 +65,14 @@
     index_date = start_date,
     # Define the study population
     ## IN AND EXCLUSION CRITERIA
+    ### (= > 1 year follow up, aged > 18 and no missings in age and sex)
+    ### missings in age are the ones > 110
+    ### missings in sex can be sex = U or sex = I (so filter on M and F)
     population=patients.satisfying(
         """
-        (age >= 18) AND 
         has_follow_up
+        AND (age >=18 AND age <= 110)
+        AND (sex = "M" OR sex = "F")
```

Comment:
I also normally include a flag in the population definition for whether a patient has died. e.g. https://github.com/opensafely/pincer-measures/blob/main/analysis/study_definition.py#L23